rskj-core/src/test/java/co/rsk/peg/BridgeSupportGetTransactionTypeTest.java

```diff
@@ -49,9 +50,25 @@ class BridgeSupportGetTransactionTypeTest {
     void getTransactionType_sentFromP2SHErpFed() {
         ActivationConfig.ForBlock activations = ActivationConfigsForTest.hop400().forBlock(0);
 
+        // should hardcode the pkeys because the redeem script size validation threw error
```

Comment:
I'd remove this comment, does add much value to the tests it's only explaining the change done here